stack/deploy.sh

```diff
@@ -1,7 +1,60 @@
 #!/bin/sh
```

Comment:
not your bug, but this looks like it relies on bashisms, not simple bourne shell.

Reply:
maybe run it past a linter somewhere?

Reply:
done!